### PR TITLE
⚡ node-fetch migration

### PR DESCRIPTION
**Pull request by dpaiv0**
self explanatory

### CODE REVIEW

File: src/apis/Chorus.js [hunk 1]
```diff
@@ -14,8 +15,7 @@ module.exports = class ChorusAPI extends APIWrapper {
   }
 
   request (endpoint, queryParams = {}) {
-    return snekfetch.get(API_URL + endpoint)
-      .query(queryParams)
-      .then(r => r.body)
+    return fetch(API_URL + endpoint + `?${qs.stringify(queryParams)}`)
```
Comment (perronosaurio) [inline src/apis/Chorus.js]: why +

Comment (dpaiv0) [reply src/apis/Chorus.js]: I've already explained this. Readability.

**Comment by dpaiv0** [comment]
I made this at school wtf

**Comment by dpaiv0** [comment]
Aight, tested this, everything works.

### RFR

**Pull request merged**